libs/blocks/preflight/checks/assets.js

```diff
@@ -22,110 +212,53 @@ export async function checkImageDimensions(url, area) {
     return JSON.parse(JSON.stringify(cachedResult));
   }
 
-  const allImages = [...area.querySelectorAll('main picture img')];
-  if (!allImages.length) {
+  const allAssets = [...area.querySelectorAll('main picture img, main video, main .adobetv')];
```

Comment:
I see the video from modal is not picked up - https://preflight-assets-updates--milo--overmyheadandbody.aem.page/docs/authoring/tall-video-modal#tvid

Reply:
Good point, I'll look into this for a follow-up contribution.